authentik/sources/ldap/sync/membership.py

```diff
@@ -53,7 +70,7 @@ def sync(self) -> int:
                         "ak_groups__in": [ak_group],
                     }
                 )
-            )
+            ).distinct()
```

Comment:
I added a distinct here because I noticed it was pulling back the same user multiple times if they were in multiple groups due to the join. It seemed wrong to call group save with the same user multiple times but Django may already be handling that internally.